nakhll/utils.py

```diff
@@ -105,3 +106,22 @@ def dict_to_table(dictionary):
     for key, value in dictionary.items():
         table += f'{key}: {value}<br>'
     return table
+
+
+
+def custom_exception_handler(exc, context):
+    # Call REST framework's default exception handler first,
+    # to get the standard error response.
+    response = exception_handler(exc, context)
+
+    # Now add the HTTP status code to the response.
+    error_message = {}
+    if response is not None:
+        error_message = {}
+        if response.data:
+            for key, value in response.data.items():
+                error_message[key] = f"{key}: {str(value[0])}"
+        error_message['status_code'] = response.status_code
+
+    response.data = error_message
+    return response
```

Comment:
please add a new line at the end of the file.
```suggestion
    return response

```